#!/usr/bin/env python3
"""
This file is created to configure some of static variables of executable scripts.
"""
main_single_arm_dic = {'CSVFileDirectory' : '/home/rebel/bimanual_ws/src/single_arm_pkg/scripts/',
                       'CSVFileName' : 'plot_data.csv',  # define name of the csv file.
                       'urdfDirectory' : '/home/rebel/bimanual_ws/src/single_arm_pkg/urdf/',
                       'urdfModelName' : 'single_6dof_arm.urdf'}

plot_csv_dic = {'CSVFileDirectory' : '/home/rebel/bimanual_ws/src/single_arm_pkg/scripts/',
                'CSVFileName' : main_single_arm_dic['CSVFileName']}
